scripts/start_webpack_dev.sh

```diff
@@ -1,3 +1,3 @@
 #!/bin/sh
 
-echo "starting webpack dev" && export NODE_OPTIONS="--max_old_space_size=4096" && yarn && rm -rf /public/packs && bin/webpack-dev-server
+echo "starting webpack dev" && export NODE_OPTIONS="--max_old_space_size=4096" && yarn --skip-integrity-check && rm -rf /public/packs && bin/webpack-dev-server
```

Comment:
is the `--skip-yarn-integrity` still needed here?

Reply:
That I do not know, lemme give it a twirl. There were literally like 4-5 diff spots to do this.

Reply:
In: e15eeb0